bip39-lib/src/jvmTest/kotlin/cash/z/ecc/android/bip39/ReadmeExamplesTestJvm.kt

```diff
@@ -0,0 +1,20 @@
+package cash.z.ecc.android.bip39
+
+import cash.z.ecc.android.bip39.Mnemonics.MnemonicCode
+import cash.z.ecc.android.bip39.Mnemonics.WordCount
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+
+class ReadmeExamplesTestJvm : ShouldSpec({
```

Comment:
Is this filename right?

Reply:
yeah the class matches the filename: `ReadmeExamplesTestJvm` 
I moved that test out because `use` is [only available on jvm](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html)

Reply:
>     * Should we make the new classes in the common and crypto packages `internal`?  I think they're implementation details and probably shouldn't be a public API.  I recognize that previously `FallbackProvider` and `Pbkdf2Sha512` were public, and I suspect that was by accident when initially implemented. Making those two internal classes would technically be a public API change.  Should we change them now?  Should we keep them all public for now, then file an issue to make them all consistently internal as a followup?

I fixed the formatting issues. The probably should be internal. But up to you, I can make them all internal if you want